packages/runtime-common/expression.ts

```diff
@@ -0,0 +1,16 @@
+import * as JSON from 'json-typescript';
+
+export type Expression = (string | Param)[];
+
+export type PgPrimitive =
+  | number
+  | string
+  | boolean
+  | JSON.Object
+  | JSON.Arr
+  | null;
+
+export interface Param {
+  param: PgPrimitive;
+  kind: 'param';
+}
```

Comment:
this will be very handy in the future when we start making sql expressions, for now we just the `PgPrimitive`